### DISCUSSION TITLE
Cannot resolve method 'as(TypeRef<GraphQLResult<Map<String, POITO>>>)'

### DISCUSSION THREAD

**Discussion by premkh9** [kobylynskyi/graphql-java-codegen#1007] [Q&A] [answered]
I am trying to use the example provided in - https://github.com/kobylynskyi/graphql-java-codegen/blob/master/plugins/maven/example-client/src/test/java/io/github/kobylynskyi/order/service/CreateProductIntegrationTest.java

I am using the latest version of Rest assured dependency (5.2.0) and I see - 'Cannot resolve method 'as(TypeRef<GraphQLResult<Map<String, POITO>>>)'

Import statement used - import io.restassured.mapper.TypeRef;

![image](https://user-images.githubusercontent.com/14880073/193996291-3e4a02c2-23a0-4502-9660-921d0eb535f7.png)

Tried to play but couldn't fix the issue. Any help would be appreciated

**Comment by kobylynskyi** [comment, marked as answer]
@premkh9 the correct import is:
`import io.restassured.common.mapper.TypeRef;`
